ACE/ace/Based_Pointer_T.h

```diff
@@ -130,6 +130,9 @@ class ACE_Based_Pointer_Basic
 
   /// Keep track of our offset from the base pointer.
   ptrdiff_t base_offset_;
+
+  static CONCRETE *based (ACE_Based_Pointer_Basic *p);
```

Comment:
Documentation?

Reply:
The macro it replaced had none -- I'll add.